### ISSUE TITLE
Aggregate class shown as attributes

### ISSUE THREAD

**Issue by gildasio** [ningirsu/idmef_parser#2]
Hi, there is some aggregate class that is shown as attributes of another class, for example `UserId` where `name` is a aggregate class but is shown as a `UserId` attribute.

![image](https://user-images.githubusercontent.com/6231305/70080012-92551e00-15e4-11ea-8e39-a58bc22de15b.png)

![image](https://user-images.githubusercontent.com/6231305/70080027-9bde8600-15e4-11ea-983d-fed7bfef89ad.png)

There are others examples like that.

Is it propurseful?

**Comment by Moutix** [comment]
Yep I took the liberty to display them the same way (here: https://github.com/ningirsu/idmef_parser/blob/master/graph_generator.rb#L86)

Basically when I did it, I supposed that this information was only useful if we want to serialize the IDMEF message as XML, and not for understanding the shema.

For instance if you want to send IDMEF message as JSON you will probably represent attributes and aggregates the same way
